common/src/main/java/org/dst/utils/enums/Status.java

```diff
@@ -0,0 +1,20 @@
+package org.dst.utils.enums;
+
+/**
+ *
+ * <p>kvstore 处理返回结果</p>

```

Comment:
In English please.

Reply:
`The status is used to describe the result of server.`

Reply:
got it!